rules_building_block/discovery_of_internet_connection_via_builtin_tools.toml

```diff
@@ -0,0 +1,50 @@
+[metadata]
+creation_date = "2023/07/11"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/07/11"
+
+[rule]
+author = ["Elastic"]
+building_block_type = "default"
+description = """
+Identifies the use of built-in tools attackers can use to check for Internet connectivity on compromised systems.
+These results may be used to determine communication capabilities with C2 servers, or to identify routes, redirectors,
+and proxy servers.
+"""
+from = "now-119m"
+interval = "60m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Discovery of Internet Capabilities via Built-in Tools"
+risk_score = 21
+rule_id = "c2d49ec7-5aaa-49e5-97bc-07faa6b1df04"
+severity = "low"
+tags = ["Domain: Endpoint", "OS: Linux", "OS: macOS", "Use Case: Threat Detection", "Tactic: Discovery", "Rule Type: BBR"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.type == "start" and event.action == "exec" and
+  process.name : ("ping")
```

Comment:
Do we really want to capture every single instance where a user uses the ping command?

Reply:
Discussed in RAD removing noisy rules.